### ISSUE TITLE
Spaces around variablenames in CSV header destroy variable recognition

### ISSUE THREAD

**Issue by Arvengeance** [Snooz82/robotframework-datadriver#83]
# Given
I have a CSV file with a header like this:
```
*** Test Cases ***;${jira};${next-value}
```

Now, since I work in VSCode, I use plugins to align the columns. My header looks then something like this:
```
*** Test Cases ***              ;${jira}    ;${next-value}
this is a really long text text ;ROBOTQA-421;foo
```

# Behaviour
When running the testcase, i get an error that 
```
Variable Variable '${jira}' not found.
```

If I remove the spaces in the header, it works.

# Expected
Since variable is already wrapped in `${}`, I would expect the spaces being trimmed.

**Comment by Arvengeance** [comment]
I don't see the good thing ;) What is the use case of keeping spaces behind the closing bracket of the variable name?

**Comment by Snooz82** [comment]
yea. you are right. I was just thinking about consistency. but i think i will just strip the headers.
But i general i would not recommend to try to align csv!!!

**Comment by Snooz82** [comment]
will be fixed in 1.7.0
